pillar/environment-prod-public.sls

```diff
@@ -32,7 +32,7 @@ journal:
     default_host: elifesciences.org
     # gtm_id: # see builder-private
     # google_optimize_id: # see builder-private
-    cookie_consent: true
+    cb_id: 0a5c50d8-fcf9-47b1-8f4f-1eaadb13941b
```

Comment:
This is fine here as the id will be present in the source code of our pages anyway.